qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -35,6 +35,22 @@
                     flags = "arch_capabilities"
                     check_guest_cmd = "cat /sys/devices/system/cpu/vulnerabilities/mds | grep '%s'"
                     expect_items = 'Not affected'
+                - test_la57:
+                    # support RHEL.8 guest or later
+                    no RHEL.6 RHEL.7
+                    type = x86_cpu_flag_la57
+                    start_vm = no
+                    flags = "la57"
+                    auto_cpu_model = no
+                    check_cmd = 'grep "address sizes" /proc/cpuinfo | uniq | grep "${bits_virtual} bits virtual"'
+                    variants:
+                        - with_la57:
+                            bits_virtual = 57
+                            cpu_model_flags = ",+la57"
+                        - without_la57:
```

Comment:
Set `cpu_model_flags = ",-la57"` and `auto_cpu_model = yes` for this case, then you don't have to determine cpu model.

Reply:
Thanks and update.